public/scripts_cr.js

```diff
@@ -0,0 +1,91 @@
+window.addEventListener("DOMContentLoaded", () => {
```

Comment:
We discussed these auth features being AJAX is not necessary, but the theory is that it is a better user experience and more performant as well. You're right. 😄  But try to limit your usage of AJAX-powered featured for this particular project.

Reply:
But user experience matter!😄 Lol, I'll try. Just really want it to be functional as much as possible.